lumps/dehacked.lmp

```diff
@@ -322,8 +322,8 @@ E1TEXT = This Site is safe now.\n\
         Sure, everyone's dead, but so are they.\n\
         The monsters that infested this place\n\
         no longer exist in this reality. But it\n\
-        feels wrong. Too simple. They never stopped\n\
-        you only slowed you down. Why?\n\n\
+        feels wrong. Too simple. They never stopped,\n\
```

Comment:
You took out the word "you" and this doesn't necessarily make sense now. They didn't stop what?

The correct text should probably be: They never stopped you, only slowed you down.